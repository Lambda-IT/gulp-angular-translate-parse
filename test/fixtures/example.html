<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title></title>
</head>
<body>
    <div translate="test_key"></div>
    <div translate="ns.test_key"></div>
    <button translate="ns1.ns2.test_key">test text</button>
    <span translate="ns1.ns3.test_key">Here there is already text inside</span>
    <p translate-html="ns1.ns3.test_html_key">This is <strong>formatted</strong> text.</BR> And another <i>line</i></p>
    <script type="text/ng-template" id="test.html">
        <div translate="template.test_key"></div>
        <button translate="template.ns2.test_key" gulp-ng-translate-preserve-whitespace='true'>test text</button>
        <span translate="template.ns3.test_key" gulp-ng-translate-preserve-whitespace >Here there is already text inside</span>
        <p translate-html="template.ns3.test_html_key">This is <strong>formatted</strong> text.</BR> And <b>another</b> <i>line</i>.</p>
    </script>
</body>
</html>
