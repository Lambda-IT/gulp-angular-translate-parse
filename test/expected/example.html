<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title/>
</head>
<body>
    <div translate="test_key"></div>
    <div translate="ns.test_key"></div>
    <button translate="ns1.ns2.test_key"></button>
    <span translate="ns1.ns3.test_key"></span>

    <script type="text/ng-template" id="test.html">
        <div translate="template.test_key"></div>
        <button translate="template.ns2.test_key"></button>
        <span translate="template.ns3.test_key"></span>
    </script>
</body>
</html>
